openslides/core/static/templates/index.html

```diff
@@ -15,92 +15,150 @@
-                                {{ gettext("Edit profile") }}</a></li>
-                            <li><a href="{% url 'password_change' %}">
-                                <span class="glyphicon glyphicon-lock" aria-hidden="true"></span>
-                                {% trans "Change password" %}</a></li>
-                            <li class="divider"></li>
-                            <li>
-                                <a ng-click="logout()">
-                                    <span class="glyphicon glyphicon-log-out" aria-hidden="true"></span>
-                                    Logout
-                                </a>
-                            </li>
-                        </ul>
-                    </div>
-                    <div ng-if="!operator.isAuthenticated()">
-                        <a href="#" ng-click="showLoginForm = true" class="btn btn-default">
-                            <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
-                            {{ gettext("Login") }}
-                        </a>
-                        <div ng-if="showLoginForm">
-                            <form>
-                                username: <input type="text" ng-model="username"><br>
-                                password: <input type="password" ng-model="password"><br>
-                                <input type="submit" ng-click="login(username, password)" value="Save" />
-                            </form>
-                        </div>
-                    </div>
+  <!-- Navbar -->
+  <nav id="header" class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a href="/" class="navbar-brand">
+          <img id="logo" src="/static/img/logo.png" alt="OpenSlides" />
+        </a>
+        <span class="navbar-text optional">{{ config('event_name') }}</span>
+      </div>
+      <div class="navbar-right" ng-controller="userMenu">
+        <!-- login/logout button -->
+        <div class="btn-group">
+          <div ng-if="operator.isAuthenticated()">
+            <button data-toggle="dropdown" class="btn btn-default dropdown-toggle">
+              <span class="glyphicon glyphicon-user" aria-hidden="true"></span>
+              <span class="optional-small">{{ operator.user.get_short_name() }}</span>
+              <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu pull-right">
+              <li><a href="{% url 'user_settings' %}">
+                    <i class="fa fa-cog"></i>
+                    <translate>Edit profile
+                  </a>
+              <li><a href="{% url 'password_change' %}">
+                    <i class="fa fa-key"></i>
+                    <translate>Change password
+                  </a>
+              <li class="divider">
+              <li><a ng-click="logout()" href="">
+                    <i class="fa fa-sign-out"></i>
+                    <translate>Logout
```

Comment:
Here is another translate element without a closing tag. The same in line 55. The best would be, if you search for ```<translate``` in your files